lib/src/utils/MuiPickersUtilsProvider.jsx

```diff
@@ -4,16 +4,24 @@ import PropTypes from 'prop-types';
 export default class MuiPickersUtilsProvider extends PureComponent {
   static propTypes = {
     utils: PropTypes.func.isRequired,
+    locale: PropTypes.oneOfType([PropTypes.object, PropTypes.string]),
     children: PropTypes.element.isRequired,
+    moment: PropTypes.func,
```

Comment:
Why moment needs to be passed to utils? Won't importing `moment` in `moment-utils` and calling `moment.locale()` do the trick?

I'm guessing it's about sharing same moment instance between `material-ui-pickers` and rest of application, but I always thought that locale is set globally per app.

Just asking to make sure we can't avoid this :)

Reply:
@cherniavskii I have tried, but i gave up. 
If you want - you can try it in this branch :)

Reply:
@cherniavskii We shouldn't set the locale globally, we cannot assume how the user is going to use this library, or with how many concurrent locales.

besides, the `locale` field isn't even used.

We have 2 options:

1. leave the locale setup to the user.
2. get the locale from the user in constructor and use the [local locale](https://momentjs.com/docs/#/i18n/instance-locale/) feature.

Reply:
@alitaheri @cherniavskii 
User can also use local locale featues by himself and pass already localed moment instance to the pickers. Because of global selecting locale is really weird in moment maybe we will merge and release this feature and then will redo it if possible :) 
Because I know that a lot of people depends on it.

Reply:
Personally, I don't like passing moment instance to Utils.
I'd rather go for option №2, proposed by @alitaheri 

That being said, releasing it "as is" sounds reasonable for me too.
It has a drawback, since changing this may be a breaking change, but it's okay at least until mui is in beta.